client/src/components/landing_page/ExploreMapSection.vue

```diff
@@ -1,26 +1,38 @@
 <template>
-  <div class='section'>
-    <div class='header-section'>
-      <div class='h1-header-large'>
-        {{ messages.EXPLORE_MAP_SECTION_HEADER }}
+  <div class='section is-small'>
+    <div class='columns is-centered has-text-centered'>
```

Comment:
Not familiar with Bulma but it looks like these properties specify a lot of the properties we are currently specifying with CSS .. if this is the case can we remove duplicate css properties? (mixins for column, center, and text-align center all seem covered here)

Reply:
Yup! Done. Good idea.